cell_classification/augmentation_pipeline.py

```diff
@@ -0,0 +1,80 @@
+import imgaug as ia
+import imgaug.augmenters as iaa
+from imgaug.augmentables.segmaps import SegmentationMapsOnImage
+from imgaug.augmentables.batches import Batch
+import numpy as np
+
+
+def augment_images(images, masks, augmentation_pipeline):
+    """
+    Augment images and masks.
+    Args:
+        images (np.array):
+            The images (b,h,w,c) to augment
+        masks (np.array):
+            The masks (b,h,w,1) to augment
+        augmentation_pipeline (imgaug.augmenters.meta.Augmenter):
+            The augmentation pipeline
+    Returns:
+        np.array:
+            The augmented images
+        np.array:
+            The augmented masks
+    """
+    masks = [SegmentationMapsOnImage(mask, shape=images.shape[1:-1]) for mask in masks]
+    batch = Batch(images=images, segmentation_maps=masks)
+    batch = augmentation_pipeline.augment_batch_(batch)
+    augmented_masks = [np.squeeze(mask.arr) for mask in batch.segmentation_maps_aug]
```

Comment:
Good to add a comment about why the shape is changing

Reply:
It doesn't change. I changed this to `shape=mask.shape` to make it more clear.